src/sdk/sdkV2.ts

```diff
@@ -36,6 +36,7 @@ export default class SdkV2 implements ISdk {
       case CHAIN_ID.MAINNET:
       case CHAIN_ID.ROPSTEN:
       case CHAIN_ID.POLYGON:
+      case CHAIN_ID.CELO:
```

Comment:
I don't think this line is necessary

Reply:
I tried to run tests without it, successful

Reply:
done, thanks man